stock_inventory_chatter/models/stock.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2019 ForgeFlow S.L. (https://www.forgeflow.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class StockInventory(models.Model):
+    _name = "stock.inventory"
+    _inherit = ["stock.inventory", "mail.thread"]
+
+    state = fields.Selection(tracking=True)
+    location_ids = fields.Many2many(tracking=True)
+    start_empty = fields.Boolean(tracking=True)
+
+    def _track_subtype(self, init_values):
+        self.ensure_one()
+        if "state" in init_values and self.state == "cancel":
```

Comment:
Looking into Odoo v13 I noticed, you can never reach to a cancel state of a Inventory Adjustment with standard core modules.